filebeat/filebeat.full.yml

```diff
@@ -211,6 +211,10 @@ filebeat.prospectors:
   # This is especially useful for multiline log messages which can get large.
   #max_bytes: 10485760
 
+  # Maximum depth for "**" patterns. When it's set to a positive value, "**"
+  # patterns are expanded up to the specified depth.
+  #recursive_glob_depth: 16
```

Comment:
Assuming we don't want to make it configurable at first, could we go with `recursive_glob.enabled` as config option instead? This will allow us to extend it in the future if needed. I'm aware that just `recursive_glob: true` would feel more natural at first, but this will prevent us from using it as a namespace in the future.